package ch12.exceptions.exercise;

/**
 * exceptions/Ex11.java
 * <p>
 * TIJ4 Chapter Exceptions, Exercise 11, page 468
 * <p>
 * ****************** Exercise 11 ******************
 * Repeat the previous exercise, but inside the catch clause, wrap g()'s
 * exception in a RuntimeException.
 * ************************************************
 *
 * @author wenwu.liu.o
 */
public class Ex11 {
}
